### PR TITLE
User signup signin signout

### PR DESCRIPTION
**Pull request by carlosm4247**
Description:
Mainly added some styling to the sign up/login forms and added a border to the bottom of the nav-bar to create that separation between it and the content. Also removed some styling for buttons which would leave an outline around them after being clicked which I didn't care too much for. 

Milestones:
User can login/logout
User can create an account
User can see their profile

Test Details:
image
image

### CODE REVIEW

File: capstone-ui/src/Components/App/App.css [hunk 1]
```diff
@@ -3,13 +3,14 @@
   max-width: 100%;
   min-height: 100vh;
   margin: 0 auto;
-  padding: 1rem;
+
```
Comment (nick-r-117) [inline capstone-ui/src/Components/App/App.css]: This looks like an extra line space. I don't think we need this.

**Review by nick-r-117 — APPROVED** [review]
Overall looks good as this is just changing the style.

One thing I think would help with review is ensure the title summarizes commits in PR. For instance, I think you used this PR's title for the UX & Core functionality and are using here for style. What you can do is: [Milestone 1] What this PR is about e.g. [User signup/signin/signout] Add styling to login/sign up forms and polish overall styling quality.

I've taken a quick look at your other PRs and they look ok. I believe the name of latest commit will transfer to PR title which will work for 1 commit PRs as you have descriptive commit messages. However, for multiple commit PRs like this one, it maybe helpful to have a title that represents the PR as a whole.

File: capstone-ui/src/Components/App/App.css (hunk 1 above)
Comment (carlosm4247) [reply capstone-ui/src/Components/App/App.css]: Good catch, I'll remove it now

**Comment by carlosm4247** [comment]
Thanks, I'll make sure the title of PRs are more descriptive in the future.

**Pull request merged**